### PR TITLE
feat : add product page

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Product Page

image

[Product Page CodeSandBox Link](https://y06v4.csb.app/pages/product/product.html)

### CODE REVIEW

**Review by Husain01 — APPROVED** [review]
I think there's a little overflow on y axis just refactor that bit. Everything else is looking good

**Review by kart-c — CHANGES_REQUESTED** [review]
Kevin, about the filter section, you should use the `name` attribute on the radio buttons so that the user is allowed only to click on one of the radio buttons (amongst a set).
Also add the `id` to the input and the same value as a `for` attribute to the labels, so that the user can still click the inputs when the label is clicked.
The bottom of the last row of grid could use some `margin-bottom` 
Also `id` selectors are supposed to be unique, you are using `card-image` on multiple `img` tags, which is not a good practice. I would suggest to replace them with a class selector

**Review by RahullRawat — APPROVED** [review]

File: pages/product/product.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  <title>eCommerce | Home</title>
+</head>
+<body>
+  <nav class='navbar'>
+    <div class="logo">
+      <h2>BlackT</h2>
+    </div>
+        
+  
+
```
Comment (VishalPatil18) [inline pages/product/product.html:19]: Multiple empty lines could be avoided.

**Review by VishalPatil18 — APPROVED** [review]
- A suggestion regarding multiple empty lines was added.
- You could decrease the size of `login` button and the `heart` and `cart` icons in the navbar. Currently, in my opinion, they are a bit large which takes my attention to them, rather than the products.

Apart from these great work @Kevin-Solomon 👍

**Pull request merged**